woocommerce/woocommerce-gutenberg-products-block
# Add Cart totals to Cart block

Fixes #1298.

#### Accessibility




### Screenshots

![imatge](https://user-images.githubusercontent.com/3616980/71086044-20aad180-2199-11ea-89e2-3e9a9a0268fe.png)

### How to test the changes in this Pull Request:

1. Create a post with a _Cart_ block.
2. Preview it in the frontend.
3. Verify it renders correctly and the button links to the checkout page.
